client/app/components/queries/QueryEditor/QueryEditorControls.jsx

```diff
@@ -43,10 +43,10 @@ export default function EditorControl({
   useEffect(() => {
     const buttons = filter(
       [addParameterButtonProps, formatButtonProps, saveButtonProps, executeButtonProps],
-      b => b.shortcut && !b.disabled && isFunction(b.onClick)
+      b => b.shortcut && (!b.disabled || b.keepShortcutKeysWhenDisabled) && isFunction(b.onClick)
```

Comment:
What if we will not add this flag, but just always keep shortcuts (with noop handler)?

Reply:
> What if we will not add this flag, but just always keep shortcuts (with noop handler)?

I considered, but at first I didn't want to risk having shortcut keys _always_ taken. But I just checked and the only control we have with both `disabled` and `shortcuts` is the execute query one, so this should be fine 👍